src/settings/MatchProfiles/MatchProfilesForm.js

```diff
@@ -109,6 +175,13 @@ MatchProfilesFormComponent.propTypes = {
   location: PropTypes.shape({ search: PropTypes.string.isRequired }).isRequired || PropTypes.string.isRequired,
   associatedJobProfilesAmount: PropTypes.number.isRequired,
   onCancel: PropTypes.func.isRequired,
+  jsonSchemas: PropTypes.shape({
+    INSTANCE: PropTypes.object,
+    HOLDINGS: PropTypes.object,
+    ITEM: PropTypes.object,
+    ORDER: PropTypes.object,
+    INVOICE: PropTypes.object,
+  }).isRequired,
```

Comment:
It can be written more simply `PropTypes.objectOf(PropTypes.object).isRequired`

Reply:
I guess this structure more accurately describes how the object should look.